plugins/tiddlywiki/tiddlyweb/tiddlywebadaptor.js

```diff
@@ -235,7 +253,16 @@ Delete a tiddler and invoke the callback with (err)
 options include:
 tiddlerInfo: the syncer's tiddlerInfo for this tiddler
 */
-TiddlyWebAdaptor.prototype.deleteTiddler = function(title,callback,options) {
+TiddlyWebAdaptor.prototype.deleteTiddler = function(title,options,callback) {
+	// Starting with 5.1.24, all syncadptor method signatures follow the node.js
+	// standard of callback as last argument. This catches the previous signature:
```

Comment:
It might be worth trying to abbreviate the repeated code here; we probably don't need the comments.